diff -ru mbedtls-mbedtls-2.24.0.orig/library/ssl_msg.c mbedtls-mbedtls-2.24.0/library/ssl_msg.c
--- mbedtls-mbedtls-2.24.0.orig/library/ssl_msg.c	2020-10-31 16:56:02.409389805 +0100
+++ mbedtls-mbedtls-2.24.0/library/ssl_msg.c	2020-10-31 16:56:24.050427146 +0100
@@ -3411,7 +3411,7 @@
 static int ssl_handle_possible_reconnect( mbedtls_ssl_context *ssl )
 {
     int ret = MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED;
-    size_t len;
+    size_t len = 0;
 
     if( ssl->conf->f_cookie_write == NULL ||
         ssl->conf->f_cookie_check == NULL )
